OpenOversight/app/main/views.py

```diff
@@ -320,6 +325,7 @@ def officer_profile(officer_id: int):
         exception_type, value, full_traceback = sys.exc_info()
         error_str = " ".join([str(exception_type), str(value), format_exc()])
         current_app.logger.error(f"Error finding officer: {error_str}")
+        abort(HTTPStatus.NOT_FOUND)
```

Comment:
I'm a little unsure of how to test this. Any advice would be appreciated, @sea-kelp @abandoned-prototype.

Reply:
I see what you're saying - it doesn't seem like this code is reachable since (as far as I can tell) the only other exception that could be thrown in the try block is [MultipleResultsFound](https://docs.sqlalchemy.org/en/14/orm/query.html#sqlalchemy.orm.Query.one), but officer id is configured as a [primary key](https://github.com/lucyparsons/OpenOversight/blob/8b12b4b954d379b82cd818b7830acd38e5546241/OpenOversight/app/models/database.py#L228). I wonder if we should just remove it.

If you do want to test it, you could patch `Query.one()` with a side effect to raise MultipleResultsFound

Reply:
Given that we are searching by the primary key, I am comfortable removing it. Thanks for the 👁️ , @sea-kelp!